packages/dds/cell/src/cell.ts

```diff
@@ -158,6 +205,13 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
         return this.data === undefined;
     }
 
+    /**
+     * {@inheritDoc ISharedCell.getAttributor}
+     */
+    public getAttributor(): CellAttributor {
+        return this.attributor;
+    }
+
```

Comment:
cannot comment as there aren't edits, but the summarization behavior for SharedCell needs to change. Right now attribution will be lost. This also needs to be tested.